### PR TITLE
Move FileInput from chakra to tailwind

### PR DESCRIPTION
**Pull request by MananTank**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
The focus of this PR is to refactor file input components for consistency and improved styling.

### Detailed summary
- Refactored `FileInput` component for consistent styling using `className` prop
- Updated classNames for better readability and maintainability
- Removed unnecessary Chakra UI imports and replaced with custom styling
- Improved file upload button and preview display functionality

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by MananTank** [comment]
* **#3926** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/3926?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MananTank and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/contract-components/contract-deploy-form/contract-metadata-fieldset.tsx [hunk 1]
```diff
@@ -41,10 +41,7 @@ export const ContractMetadataFieldset: React.FC<
                   shouldTouch: true,
                 })
               }
-              border="1px solid"
-              borderColor="gray.200"
-              borderRadius="md"
-              transition="all 200ms ease"
+              className="border rounded transition-all duration-200"
```
Comment (jnsdls) [inline apps/dashboard/src/components/contract-components/contract-deploy-form/contract-metadata-fieldset.tsx:44]: border-border to make sure the color is right when used inside pages?

**Review by jnsdls — APPROVED** [review]
Overall looks good, might need border-border in all places though?

File: apps/dashboard/src/components/contract-components/contract-deploy-form/contract-metadata-fieldset.tsx (hunk 1 above)
Comment (MananTank) [reply apps/dashboard/src/components/contract-components/contract-deploy-form/contract-metadata-fieldset.tsx:44]: good catch, added

**Pull request merged**